src/components/Typography.tsx

```diff
@@ -28,17 +30,32 @@ export const PageHeading: FunctionComponent = ({
   <h1 className="govuk-heading-l">{children}</h1>
 );
 
-export const SectionHeading: FunctionComponent = ({
+export const SectionHeading: FunctionComponent<SectionHeadingProps> = ({
   children,
+  classes,
 }: SectionHeadingProps): JSX.Element => (
-  <h2 className="govuk-heading-m">{children}</h2>
+  <h2 className={"govuk-heading-m " + classes || ""}>{children}</h2>
 );
 
 export const AnchorLink: FunctionComponent<AnchorLinkProps> = ({
   href,
   children,
+  classes,
 }: AnchorLinkProps): JSX.Element => (
-  <a href={href} className="govuk-link">
+  <a href={href} className={"govuk-link " + classes || ""}>
+    {children}
+  </a>
+);
+
+export const WarningLink: FunctionComponent<AnchorLinkProps> = ({
+  href,
+  children,
+}: AnchorLinkProps): JSX.Element => (
+  <a
+    href={href}
+    className="govuk-link govuk-link--no-visited-state"
+    style={{ color: "#d4351c" }}
+  >
     {children}
   </a>
```

Comment:
See [GDS colours](https://design-system.service.gov.uk/styles/colour/).  There's a SASS variable in there for this colour, but I couldn't work out how to get it to work here and I didn't want to spent a great deal of time on it.  So I've used the literal colour instead.